### ISSUE TITLE
android issue related property zIndex of style

### ISSUE THREAD

**Issue by andriy-shakh** [czystyl/react-native-reanimated-flip#3]
in your style:
```
const styles = StyleSheet.create({
  container: {
    justifyContent: "center",
    alignItems: "center",
  },
  side: {
    width: "100%",
    height: "100%",
    position: "absolute",
  },
  front: {
    zIndex: 2,
  },
  back: {
    zIndex: 1,
  },
})
```

the property "zIndex" affects the client code if the back card has a button. it can't handle the click on the back of the card. if I have a button on the front card, it works fine. iOS works fine. if I delete next code with your package, it works fine for android (it handle click of a button on the back card) 

```
 front: {
    zIndex: 2,
  },
  back: {
    zIndex: 1,
  },
```

if there a reason to keep these zIndex properties or maybe I can control this style from my client code?

**Comment by czystyl** [comment]
I think we can remove these two styles because they were used in the first iteration. 

PR appreciated for this :)

**Comment by andriy-shakh** [comment]
should I prepare PR for this?

**Comment by czystyl** [comment]
@andriy-shakh if you can then would be great 🙌

**Comment by czystyl** [comment]
@andriy-shakh it should be fixed by this PR #8
